chall01/spentti.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+import sys
+import re
```

Comment:
I see `re` but can't see any regex in your code. Make sure not to import a library you are not using.